docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,40 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier, and faster!
+
+### How to use template repositories for an assignment?
+
+
+![Searches for repository, then selects the option of Importing starter code with template repositories](https://user-images.githubusercontent.com/1490434/61005360-10631080-a336-11e9-843b-ddb3e4f011ca.gif)
+
+
+When you pick your template repository as a starter code repository, you will now get two options:
+
+  * Import starter code using a template repository.
+  * Import starter code using source importer.
+  * After selecting either of the options, the following steps are the same as before. More information on this can be found on our [Videos page](/videos)
```

Comment:
This shouldn't be in a bullet point. Looks a little weird since you're saying "two options" and then have three bullet points.
> the following steps are the same as before
This might not be the best choice of words since you're assuming the people reading this are existing users. New users will definitely be reading this too.

Reply:
Oops, i thought i fixed that. Good catch!